### PR TITLE
send CUSTOM_SUCCESS_MESSAGE in the second message

### PR DESCRIPTION
**Pull request by Dex-Spirit**

### CODE REVIEW

File: src/classes/MyHandler.ts [hunk 1]
```diff
@@ -1636,12 +1636,10 @@ export = class MyHandler extends Handler {
...
                 if (offer.state === TradeOfferManager.ETradeOfferState.Accepted) {
-                    this.bot.sendMessage(
-                        offer.partner,
-                        process.env.CUSTOM_SUCCESS_MESSAGE
-                            ? process.env.CUSTOM_SUCCESS_MESSAGE
-                            : '/pre ✅ Success! The offer went through successfully.'
-                    );
+                    this.bot.sendMessage(offer.partner, '/pre ✅ Success! The offer went through successfully.');
+                    if (process.env.CUSTOM_SUCCESS_MESSAGE) {
+                        this.bot.sendMessage(offer.partner, `${process.env.CUSTOM_SUCCESS_MESSAGE}`);
```
Comment (idinium96) [inline src/classes/MyHandler.ts:1641]: no need to make it in template literals form. Just make it like:
```this.bot.sendMessage(offer.partner, process.env.CUSTOM_SUCCESS_MESSAGE);```

**Comment by idinium96** [comment]
🤔

File: src/classes/MyHandler.ts (hunk 1 above)
Comment (Bonfire) [reply src/classes/MyHandler.ts:1641]: Curly braces here can be removed for brevity if desired. Resultant code is shown below.

```
if (process.env.CUSTOM_SUCCESS_MESSAGE)
     this.bot.sendMessage(offer.partner, `${process.env.CUSTOM_SUCCESS_MESSAGE}`);
```

Comment (idinium96) [reply src/classes/MyHandler.ts:1641]: no... it against ESLint and Prettier rules.

**Pull request closed without merging**